packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -752,6 +760,9 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             0x0ec /* "Unexpected value for previously processed message's sequence number" */);
 
         for (const message of messages) {
+            if (message.type === MessageType.NoOp){
```

Comment:
I do not think this is the right place to do it. I think (if I got it right) you are trying to capture all the ops we are sending. If so, this should be in submit() method.

This function deals with inbounding of ops, and it deals with gaps in op stream, as well as duplicated ops. So any logic here will very likely be incorrect, as it may count same op twice (if same op shows up twice).